src/workspaces/ShareWorkspaceModal/Collaborator.tsx

```diff
@@ -114,7 +114,7 @@ export const AclInput: React.FC<AclInputProps> = (props: AclInputProps) => {
               ...Utils.switchCase(
                 o?.value,
                 ['READER', () => ({ canCompute: false, canShare: false })],
-                ['WRITER', () => ({ canCompute: !isAzureWorkspace, canShare: false })],
+                ['WRITER', () => ({ canCompute: hasAccessLevel('OWNER', maxAccessLevel), canShare: false })],
```

Comment:
For Azure, we don't show the `canCompute` or `canShare` options at all…. so I'm not sure how this crept in.